R/export_cdisc.R

```diff
@@ -93,19 +100,35 @@ export_cdisc <- function(res_nca) {
     #  Recode PPTESTCD PKNCA names to CDISC abbreviations
     mutate(
       PPTESTCD = translate_terms(PPTESTCD, mapping_col = "PKNCA", target_col = "PPTESTCD"),
+      PPTEST = translate_terms(PPTESTCD, mapping_col = "PPTESTCD", target_col = "PPTEST"),
       DOMAIN = "PP",
       # Group ID
-      PPGRPID =  paste(ANALYTE, PCSPEC, paste("CYCLE", DOSNO,  sep = " "), sep = "-"),
+      PPGRPID = {
+        if ("PCGRPID" %in% names(.)) PCGRPID
+        else if ("AVISIT" %in% names(.)) paste(ANALYTE, PCSPEC, AVISIT, sep = "-")
+        else if ("VISIT" %in% names(.)) paste(ANALYTE, PCSPEC, VISIT, sep = "-")
+        else paste(ANALYTE, PCSPEC, DOSNO, sep = "-")
+      },
```

Comment:
```suggestion
      PPGRPID = {
        if ("AVISIT" %in% names(.)) paste(ANALYTE, PCSPEC, AVISIT, sep = "-")
        else if ("VISIT" %in% names(.)) paste(ANALYTE, PCSPEC, VISIT, sep = "-")
        else paste(ANALYTE, PCSPEC, DOSNO, sep = "-")
      },
```
PCGRPID not needed, it really not the same format